src/store/account-details/account-details.thunks.ts

```diff
@@ -128,7 +128,10 @@ function fetchHistoryTransactions(
       accountDetails: { historyTransactionsTask },
     } = getState();
 
-    if (fromItem === undefined && historyTransactionsTask.status === "successful") {
+    if (
+      fromItem === undefined &&
+      ["reloading", "successful"].includes(historyTransactionsTask.status)
```

Comment:
I would keep using the other pattern just for consistency and readability even if this is more clever